params.go

```diff
@@ -346,6 +346,12 @@ var (
 		"Length of the term for the Judge",
 	)
 
+	iigoDefaultSanctionScore = flag.Uint(
```

Comment:
this implies that the sanctionscore can be changed mid game right

Reply:
Yup so judges have the option to upload their own custom scoring for particular rules that they thing should be scored more/less severely than the default. This value is what we revert to if the client judge doesn't give us a custom value for a particular rule.